### PR TITLE
Add KeeAnywhere (keepass-plugin-keeanywhere) version 1.5.1

### PR DESCRIPTION
**Pull request by crheidri**
Added support for KeeAnywhere Keepass plugin.

### CODE REVIEW

File: bucket/keepass-plugin-keeanywhere.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "homepage": "https://github.com/Kyrodan/KeeAnywhere",
+    "description": "KeeAnywhere is a KeePass plugin that provides access to cloud storage providers (cloud drives). The main goal is to offer a simple UI while integrating deeply into KeePass. This covers all kind of users: novices, experienced and power users.",
+    "license": "MIT License",
```
Comment (Ash258) [inline bucket/keepass-plugin-keeanywhere.json]:
```suggestion
    "license": "MIT",
```

File: bucket/keepass-plugin-keeanywhere.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "homepage": "https://github.com/Kyrodan/KeeAnywhere",
+    "description": "KeeAnywhere is a KeePass plugin that provides access to cloud storage providers (cloud drives). The main goal is to offer a simple UI while integrating deeply into KeePass. This covers all kind of users: novices, experienced and power users.",
```
Comment (Ash258) [inline bucket/keepass-plugin-keeanywhere.json]:
```suggestion
    "description": "KeePass plugin that provides access to cloud storage providers (cloud drives).",
```

File: bucket/keepass-plugin-keeanywhere.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "version": "1.5.1",
+    "url": "https://github.com/Kyrodan/KeeAnywhere/releases/download/v1.5.1/KeeAnywhere-1.5.1.plgx",
+    "hash": "bf7608ff3032dd1621b4c66e7c34a7f7c3357cb436efe0e50613e01e319be2da",
+    "depends": "extras/keepass",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/Kyrodan/KeeAnywhere/releases/download/v$version/KeeAnywhere-$version.plgx"
+    },
+    "installer": {
+        "script": "Copy-Item \"$dir\\KeeAnywhere-$version.plgx\" \"$(appdir keepass $global)\\current\\Plugins\" -Force"
```
Comment (Ash258) [inline bucket/keepass-plugin-keeanywhere.json]:
```suggestion
        "script": "Copy-Item \"$dir\\$fname\" \"$(appdir keepass $global)\\current\\Plugins\" -Force"
```

File: bucket/keepass-plugin-keeanywhere.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    },
+    "uninstaller": {
+        "script": "Remove-Item \"$(appdir keepass $global)\\current\\Plugins\\KeeAnywhere-$version.plgx\""
```
Comment (Ash258) [inline bucket/keepass-plugin-keeanywhere.json]:
```suggestion
        "script": "Remove-Item \"$(appdir keepass $global)\\current\\Plugins\\$fname\""
```

**Comment by r15ch13** [comment]
The `plgx` file can be renamed while downloading. No version in the uninstaller necessary.

```json
{
    "homepage": "https://github.com/Kyrodan/KeeAnywhere",
    "description": "KeePass plugin that provides access to cloud storage providers (cloud drives).",
    "license": "MIT",
    "version": "1.5.1",
    "url": "https://github.com/Kyrodan/KeeAnywhere/releases/download/v1.5.1/KeeAnywhere-1.5.1.plgx#/KeeAnywhere.plgx",
    "hash": "bf7608ff3032dd1621b4c66e7c34a7f7c3357cb436efe0e50613e01e319be2da",
    "depends": "extras/keepass",
    "checkver": "github",
    "autoupdate": {
        "url": "https://github.com/Kyrodan/KeeAnywhere/releases/download/v$version/KeeAnywhere-$version.plgx#/KeeAnywhere.plgx"
    },
    "installer": {
        "script": "Copy-Item \"$dir\\KeeAnywhere.plgx\" \"$(appdir keepass $global)\\current\\Plugins\" -Force"
    },
    "uninstaller": {
        "script": "Remove-Item \"$(appdir keepass $global)\\current\\Plugins\\KeeAnywhere.plgx\""
    },
    "notes": "Amazon Drive: Due to unclear Amazon Policy this Provider may stop working at any time. Please do not rely on this functionality!"
}
```

**Comment by crheidri** [comment]
> The `plgx` file can be renamed while downloading. No version in the uninstaller necessary.
> 
> ```json
> {
>     "homepage": "https://github.com/Kyrodan/KeeAnywhere",
>     "description": "KeePass plugin that provides access to cloud storage providers (cloud drives).",
>     "license": "MIT",
>     "version": "1.5.1",
>     "url": "https://github.com/Kyrodan/KeeAnywhere/releases/download/v1.5.1/KeeAnywhere-1.5.1.plgx#/KeeAnywhere.plgx",
>     "hash": "bf7608ff3032dd1621b4c66e7c34a7f7c3357cb436efe0e50613e01e319be2da",
>     "depends": "extras/keepass",
>     "checkver": "github",
>     "autoupdate": {
>         "url": "https://github.com/Kyrodan/KeeAnywhere/releases/download/v$version/KeeAnywhere-$version.plgx#/KeeAnywhere.plgx"
>     },
>     "installer": {
>         "script": "Copy-Item \"$dir\\KeeAnywhere.plgx\" \"$(appdir keepass $global)\\current\\Plugins\" -Force"
>     },
>     "uninstaller": {
>         "script": "Remove-Item \"$(appdir keepass $global)\\current\\Plugins\\KeeAnywhere.plgx\""
>     },
>     "notes": "Amazon Drive: Due to unclear Amazon Policy this Provider may stop working at any time. Please do not rely on this functionality!"
> }
> ```

This is the cleanest solution. See 1d4101d12b438e618f3513bb19c70760f527c4ff. Autoupdate tested and working with `checkver.ps1`.

**Pull request merged**